app/src/main/java/pl/example/android/garbageapp/data/network/SectorTermsNetworkDataSource.java

```diff
@@ -98,27 +101,34 @@ public void startSectorTermsSyncService() {
     public void fetchSectorTerms() {
         Log.d(LOG_TAG, "Fetch sector terms started");
         DatabaseReference blueRef = mFirebaseDatabaseReference.child("blue");
-        blueRef.addValueEventListener(new SectorValueEventListener());
+        blueRef.addValueEventListener(new SectorValueEventListener(SectorType.BLUE));
 
         DatabaseReference greenRef = mFirebaseDatabaseReference.child("green");
-        greenRef.addValueEventListener(new SectorValueEventListener());
+        greenRef.addValueEventListener(new SectorValueEventListener(SectorType.GREEN));
 
         DatabaseReference yellowRef = mFirebaseDatabaseReference.child("yellow");
-        yellowRef.addValueEventListener(new SectorValueEventListener());
+        yellowRef.addValueEventListener(new SectorValueEventListener(SectorType.YELLOW));
     }
 
-    public LiveData<List<Sector>> getDownloadedSectors() {
+    public LiveData<List<SectorTerm>> getDownloadedSectors() {
         return mDownloadedSectors;
     }
 
     private class SectorValueEventListener implements ValueEventListener {
 
+        private SectorType sectorType;
+
+        SectorValueEventListener(SectorType sectorType){
+            super();
+            this.sectorType = sectorType;
+        }
+
         @Override
         public void onDataChange(DataSnapshot dataSnapshot) {
-            List<Sector> sectorModel = new ArrayList<>();
+            List<SectorTerm> sectorModel = new ArrayList<>();
```

Comment:
Change name of the variable: sectorModel --> sectorTerms.

Reply:
OK